docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`2876` App update experience is improved when running on electron.
```

Comment:
how is the experience updated? It's not clear from the changelog. Please explain in a few more words.